changelog.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 2.7.0~rc2
+
+### Web applications
+
+* Update translation workflow to support continuous updates (#7010)
+
+### Operations
+
+* Use systemd timers to check for disconnected submissions and source listings (#7009)
```

Comment:
Optional nit: original #7009 → backport #7015.

Reply:
In general IMO it's more useful to reference the original PR rather than the backport.

Reply:
I agree; that's why I flagged <https://github.com/freedomofpress/securedrop/pull/7018#discussion_r1366209819>.

Reply:
d'oh